modules/app.xql

```diff
@@ -59,6 +61,12 @@ declare function app:scheduler-enabled($node as node(), $model as map(*)) {
         $node
 };
 
+declare 
+    %templates:warp
+function app:java-version ($version as xs:string?, $model as map(*)) as node()? {
```

Comment:
are you asking why i called it ``$version`` or why there is a first parameter? i thought giving expressive parameter names was good practice, how about ``$str``?